docs/source/_static/switcher.json

```diff
@@ -4,6 +4,10 @@
         "version": "main",
         "url":"https://umami-hep.github.io/puma/main/"
     },
+    {
+        "version": "jobirk-docs-improvements",
```

Comment:
you want to keep this in the docs?

I would rather go to version `v0.1.1` directly now

Reply:
sorry - I forgot to remove this after testing ...

I'd say we only add the docs in the PR and then update the tag + docs afterwards